tests/test_basic_regression.py

```diff
@@ -3,6 +3,7 @@
 import os
 import re
 from pathlib import Path
+from io import StringIO
```

Comment:
unused

Reply:
Oops sorry about that, also found another careless mistake, should be fixed in dd7434f